### PR TITLE
Fix plaid battery sensor

### PR DESCRIPTION
**Pull request by Adminiuga**
Fix plaid battery sensor.

PS:  and that's why you write the tests, kids :) Next time

### CODE REVIEW

**Comment by Adminiuga** [comment]
Fixes https://github.com/zigpy/zha-device-handlers/issues/366

File: zhaquirks/plaid/soil.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class PowerConfigurationClusterMains(PowerConfigurationCluster):
     def _update_attribute(self, attrid, value):
         super()._update_attribute(attrid, value)
         if attrid == self.MAINS_VOLTAGE_ATTR:
-            super()._update_attribute(self.self.BATTERY_VOLTAGE_ATTR, value)
+            super()._update_attribute(self.BATTERY_VOLTAGE_ATTR, round(value / 100))
```
Comment (dmulcahey) [inline zhaquirks/plaid/soil.py:28]: Doesn’t the rounding already happen in HA?

Comment (dmulcahey) [reply zhaquirks/plaid/soil.py:28]: Nvm... missed what was rounded

**Comment by Adminiuga** [comment]
meh, it  would need an approach similar to Eurotronic TRV valve...

**Pull request merged**